src/image/loading_displaying.js

```diff
@@ -16,55 +16,78 @@ import '../core/friendly_errors/file_errors';
 import '../core/friendly_errors/fes_core';
 
 /**
- * Loads an image from a path and creates a <a href="#/p5.Image">p5.Image</a> from it.
+ * Loads an image to create a <a href="#/p5.Image">p5.Image</a> object.
  *
- * The image may not be immediately available for rendering.
- * If you want to ensure that the image is ready before doing
- * anything with it, place the <a href="#/p5/loadImage">loadImage()</a> call in <a href="#/p5/preload">preload()</a>.
- * You may also supply a callback function to handle the image when it's ready.
+ * `loadImage()` interprets the first parameter one of two ways. If the path
```

Comment:
Perhaps this could be:
"interprets the first parameter in one of several ways."
 1) Path to local file, 2) URL, 3) base64 raw image data.

Reply:
Good suggestion–fixed!